scripts/mkcloud

```diff
@@ -798,10 +802,16 @@ function prepareinstcrowbar()
     return $?
 }
 
+function scp_install_chef_suse_override()
+{
+    if [ -e "$install_chef_suse_override" ]; then
```

Comment:
I suppose we could make $install_chef_suse_override default to ./install-chef-suse.sh. I don't personally like that behaviour, but I can tolerate it if I can override it.